middleware/validation.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file Validation middleware that checks requests using Joi schemas
+ * @author GTS
+ */
+
+import { authSchema, createUserSchema, updateUserSchema } from "./validationSchemas.js";
+
+/**
```

Comment:
Really nice documentation here, and on the rest of the validation. It's really helpful to figure out whats going on.